config/remote/remote.go

```diff
@@ -0,0 +1,224 @@
+package remote
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+	"sync"
+
+	"github.com/NordSecurity/nordvpn-linux/config"
+	"github.com/NordSecurity/nordvpn-linux/internal"
+)
+
+type RemoteStorage interface {
+	GetRemoteFile(name string) ([]byte, error)
+}
+
+// ConfigGetter get values from remote config
+type ConfigGetter interface {
+	GetTelioConfig() (string, error)
+	FeatureConfig
+}
+
+type FeatureConfig interface {
+	IsFeatureEnabled(featureName string) bool
+	GetFeatureParam(featureName, paramName string) (string, error)
+}
+
+type ConfigLoader interface {
+	LoadConfig() error
+}
+
+const (
+	envUseLocalConfig = "USE_LOCAL_CONFIG"
```

Comment:
Maybe it would be safer to have `NORDVPN_` prefix here to avoid conflicts if something else uses such env var